backends/core/models.py

```diff
@@ -59,6 +61,13 @@ class Pipeline(BaseModel):
 
   def __init__(self, name=None):
     self.name = name
+    cached_values = {
+      "failed_jobs": 0, 
+      "remaining_jobs": len(self.jobs.all()), 
```

Comment:
Small tip: pep8 will complain because single quotes are used in this file. We shouldn't mix single quotes with double quotes.

Reply:
done